.github/workflows/Build.yml

```diff
@@ -0,0 +1,72 @@
+---
+name: Build
+on:
+  workflow_call:
+    inputs:
+      packages:
+        required: true
+        type: string
+jobs:
+  build_x86_64:
+    runs-on: ubuntu-latest
+    container:
+      # This is using an older version of the dockerfile to avoid issues with actions/upload-artifact
+      image: satmandu/crewbuild:amd64@sha256:8ce37cddc8950a6717c2b6a335580aa4142393926fff0640ec64b6b78ee530b8
+      options: -u chronos
+    steps:
+      - name: Setup environment
+        run: |
+          mkdir /usr/local/manifests
+          mkdir /usr/local/build
+      - name: Build package
+        run: |
+          export CREW_LOCAL_MANIFEST_PATH=/usr/local/manifests
```

Comment:
Not FHS3 compliant?

Reply:
FHS3?

Reply:
https://refspecs.linuxfoundation.org/FHS_3.0/index.html

Putting stuff under /usr/local that shouldn't be there...

Reply:
It's only there for the life of the workflow, and its an easily accessible and writable directory on ChromeOS.